src/IB/IBExplicitHierarchyIntegrator.cpp

```diff
@@ -156,9 +156,10 @@ IBExplicitHierarchyIntegrator::preprocessIntegrateHierarchy(const double current
 } // preprocessIntegrateHierarchy
 
 void
-IBExplicitHierarchyIntegrator::integrateHierarchy(const double current_time, const double new_time, const int cycle_num)
+IBExplicitHierarchyIntegrator::integrateHierarchySpecialized(const double current_time,
+                                                             const double new_time,
+                                                             const int cycle_num)
 {
-    IBHierarchyIntegrator::integrateHierarchy(current_time, new_time, cycle_num);
```

Comment:
I'm actually surprised that this works. We should definitely still call `IBHierarchyIntegrator::integrateHierarchySpecialized()` here even if that function doesn't do anything.

Reply:
I added an empty implementation of `HierarchyIntegrator::integrateHierarchySpecialized()` so we can make this call.

Note the only integrator that doesn't call it's immediate base class is `BrinkmanAdvDiffSemiImplicitHierarchyIntegrator`, which inherits from `AdvDiffSemiImplicitHierarchyIntegrator` but calls `AdvDiffHierarchyIntegrator::integrateHierarchySpecialized()`. This is consistent with the class's behavior prior to this PR.